browser/BUILD.gn

```diff
@@ -129,6 +131,7 @@ source_set("browser_process") {
     "//components/safe_browsing/common:safe_browsing_prefs",
     "//components/search_engines",
     "//components/spellcheck/browser",
+    "//components/strings",
```

Comment:
doesn't seem to be any direct dependency added here? Or maybe it's just hidden by default in github so my search comes up empty?